### PR TITLE
Add quickstart to readme

### PR DESCRIPTION
**Pull request by juanitorduz**
- Adding quickstart and some plots to make the README more engaging. 
- Added some additional features for the mmm module.
- Added PyMC Logo closer to the start (and link to contact and services)
- Community links

----
📚 Documentation preview 📚: https://pymc-marketing--653.org.readthedocs.build/en/653/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +56,54 @@ We provide a `Dockerfile` to build a Docker image for PyMC-Marketing so that is
...
+- **Out-of-sample Predictions**: Forecast future marketing performance with credible intervals. Use this for simulations and scenario planning.
 - **Budget Optimization**: Allocate your marketing spend efficiently across various channels for maximum ROI.
 - **Experiment Calibration**: Fine-tune your model based on empirical experiments for a more unified view of marketing.
 
+### MMM Quickstart
+
+```python
+import pandas as pd
+from pymc_marketing.mmm import DelayedSaturatedMMM
+
```
Comment (wd60622) [inline README.md:72]: Nitpick: prefer the look of single space between inport and code in readme file 😅

File: README.md [hunk 2]
```diff
@@ -72,6 +132,30 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
...
+beta_geo_model.fit()
+```
+
+Once fitted, we can use the model to predict the number of future purchases for known customers, the probability that they are still alive, and get various visualizations plotted.
+
+![](/docs/source/_static/expected_purchases.png)
+
+See the Examples section for more on this.
+
 ----
```
Comment (wd60622) [inline README.md:159]: Break not need imo with the section below

**Review by wd60622 — APPROVED** [review]
Small comments but looks good

File: README.md (hunk 2 above)
Comment (juanitorduz) [reply README.md]: ok! Fixed in https://github.com/pymc-labs/pymc-marketing/pull/653/commits/489510c17611fb4424ec759a980d1f21fbff1e8d

File: README.md (hunk 1 above)
Comment (juanitorduz) [reply README.md:72]: right! fixed in https://github.com/pymc-labs/pymc-marketing/pull/653/commits/489510c17611fb4424ec759a980d1f21fbff1e8d

**Pull request merged**